packages/rancher-monitoring/prometheus-federator/package.yaml

```diff
@@ -1,4 +1,4 @@
 url: https://github.com/rancher/prometheus-federator.git
 subdirectory: charts/prometheus-federator/0.3.3
 commit: 002944bfa164dc325613e2acd9a00e9014799297
-version: 3.0.0
+version: 3.0.1
```

Comment:
can we also bump the version of `rancher-project-monitoring` (even though its a subchart) since the annotation change was made, but was never bumped.

Reply:
Also making sure to revert the change made at https://github.com/rancher/charts/blob/dev-v2.7.11/charts/rancher-project-monitoring/3.0.0%2Bup0.3.3/Chart.yaml#L5 (permalink)

Reply:
ops good catches thanks!